modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss

```diff
@@ -60,6 +60,11 @@
 		border-radius: 8px;
 	}
 
+	&-item--danger.treeview-link {
+		background-color: transparent;
+		color: #da1414 !important;
```

Comment:
could you change this to the variable defined by frontend infra `$danger`? you only need to import `@import 'atlas-variables';` at the start of the file and you are ready to go!

Reply:
Done 😄